Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -275,48 +275,6 @@ def check_python_symlinks(name, keg_only)
     "Python formulae that are keg-only should not create `pip3` and `wheel3` symlinks."
   end
 
-  INVALID_EMPTY_FILE_NAMES = %w[AUTHORS ChangeLog changes COPYING LICENSE NEWS README TODO].freeze
-  INVALID_EMPTY_FILE_EXTENSIONS = %w[.html .md .rst .txt].freeze
-  INVALID_EMPTY_FILE_DIRECTORIES = %w[bin libexec share/man share/doc].freeze
```

Comment:
I think we can just remove `libexec` here and keep the rest. I think not having empty files in `bin` and `man`/`share` is still useful.

Of course, I'm not sure it's useful enough to justify carrying around this much code for it.

Reply:
@carlocab If we don't need these files: I'd suggest we just auto delete them from `bin` and `share` if we're sure it's not going to break things. I'd still probably rather just revert this for now due to multiple maintainers complaining.